### PR TITLE
Feature/mapbox

### PR DESCRIPTION
**Pull request by robertshum**
Throttling to 5 properties for now.

Doesn't handle duplicated entries.  (if duplicate, just skip)

Does not have a modal popup.

Has input box that can change map view to a specific postal code.

For sure if we wanted to pursue this feature more, we would make the API call to geonames on property creation, then store it as part of our model.

### CODE REVIEW

File: frontend/src/routes/Map.jsx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+                markersArray.push({
+                  [postalCode]: {
+                    streetName: property.streetName,
+                    streetNumber: property.streetNumber,
+                    long: lng,
+                    lat: lat,
+                    firstName: property.landlordId.firstName,
+                    lastName: property.landlordId.lastName,
+                  }
```
Comment (garrickhenne) [inline frontend/src/routes/Map.jsx:123]: May be beneficial to include landlords id as well to use as a direct entry to landlords dashboard for modal

```suggestion
                    streetName: property.streetName,
                    streetNumber: property.streetNumber,
                    long: lng,
                    lat: lat,
                    firstName: property.landlordId.firstName,
                    lastName: property.landlordId.lastName,
                    landlordId: property.landlordId._id
                  }
```

**Review by garrickhenne — APPROVED** [review]
Looks swell to me! As another enhancement in the future it would be nice to move geonames API to get called when a new property is created and add latlong to the property document so we can reduce the calls in the front end

File: frontend/src/routes/Map.jsx (hunk 1 above)
Comment (robertshum) [reply frontend/src/routes/Map.jsx:124]: great catch

**Review by garrickhenne — APPROVED** [review]
Merge on, sir

**Pull request merged**